hoomd/mpcd/ReverseNonequilibriumShearFlow.cc

```diff
@@ -0,0 +1,418 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
```

Comment:
You will also need to run pre-commit after merging to update the license headers.